### DISCUSSION TITLE
eme_events Limit not working

### DISCUSSION THREAD

**Discussion by reditechoz** [liedekef/events-made-easy#622] [General]
This is the shortcode that is on my webpage: [eme_events limit=4]

However it is displaying all events.  See https://andrewredfern.com/speaker/

Any help greatly appreciated.

**Comment by liedekef** [comment]
limit=4 should work perfectly, I tested this on many pages (even just now). Is there no filter in place?

**Comment by reditechoz** [comment]
Yes, it seems that it not being filtered out.

Am wondering if it is something to do with Divi which is what I have built
the site in?

**Comment by liedekef** [comment]
I meant if you created a filter of your own that might influence this, but based on your answer I guess not.
The builder/theme/... is not relevant here. I can't say more in this case, except that debug needs to be done if it is actually like you said. So: either test that other settings for the list-shortcode are taken into account, or give me a temp admin access so I can do some digging myself.
